testsuite/elytron/src/test/resources/logging.properties

```diff
@@ -0,0 +1,31 @@
+# Additional loggers to configure (the root logger is always configured)
+loggers=org.wildfly.security,org.jboss.remoting3
+
+logger.level=INFO
+logger.handlers=FILE,CONSOLE
+
+logger.org.wildfly.security.level=INFO
+logger.org.wildfly.security.useParentHandlers=true
+
+logger.org.jboss.remoting3.level=INFO
+logger.org.jboss.remoting3.useParentHandlers=true
+
+handler.FILE=org.jboss.logmanager.handlers.PeriodicRotatingFileHandler
+handler.FILE.level=ALL
```

Comment:
never in default config.
default level for testsuite could only be OFF not ALL.